pkg/cmd/cli/cmd/set/triggers.go

```diff
@@ -512,6 +522,44 @@ func defaultNamespace(namespace, defaultNamespace string) string {
 	return namespace
 }
 
+// NewAnnotationTriggers creates a trigger definition from an object that can be triggered by the image
+// annotation.
+func NewAnnotationTriggers(obj runtime.Object) (*TriggerDefinition, error) {
+	m, err := meta.Accessor(obj)
+	if err != nil {
+		return nil, err
+	}
+
+	t := &TriggerDefinition{ConfigChange: true}
+	switch typed := obj.(type) {
+	case *kextensions.Deployment:
+		t.ConfigChange = !typed.Spec.Paused
```

Comment:
For deployments though this is how it behaves, so I think it's correct for Deployments and DC will remain as is.